### PR TITLE
News on ELIXIR annual report 2022 (review/Galaxy perspective)

### PR DESCRIPTION
**Pull request by sebastian-schaaf**
ELIXIR's annual report on the year 2022 has been published in the summer; this blog post reviews it from the Galaxy perspective.
I am not sure about the date to be set for this blog post; the report's publication is some time ago, but a recent date for the blog post (-> metadata field) would create visibility.

### CODE REVIEW

File: content/news/2023-06-24-elixir-annual-report-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: 'ELIXIR releases 2022 Annual Report - the Galaxy perspective'
+date: "2023-11-17"
```
Comment (bgruening) [inline content/news/2023-06-24-elixir-annual-report-2022/index.md:3]: Shouldn't this be also June or something?

Comment (sebastian-schaaf) [reply content/news/2023-06-24-elixir-annual-report-2022/index.md:3]: This was exactly my question (be strict or enhance visibility)

Comment (bgruening) [reply content/news/2023-06-24-elixir-annual-report-2022/index.md:3]: Visibility is gone, this is old news. We have been to slow. So this is about linking in reports and getting a (hopefully comprehensive) overview about the year imho.

Comment (sebastian-schaaf) [reply content/news/2023-06-24-elixir-annual-report-2022/index.md]: Ok - has been changed. I hope for a sufficient degree of comprehensiveness, too.

**Review by bgruening — APPROVED** [review]
Cool, thanks @sebastian-schaaf

**Pull request merged**